gms/api/src/main/pegasus/com/linkedin/ml/MLModelKey.pdl

```diff
@@ -0,0 +1,30 @@
+namespace com.linkedin.ml
+
+import com.linkedin.common.DataPlatformUrn
+import com.linkedin.common.FabricType
+
+/**
+ * Key for MLModel resource
+ */
+record MLModelKey {
+
+  /**
+   * Standardized platform urn where ML Model is defined. The data platform Urn (urn:li:platform:{dataScienceplatform_name})
+   */
+  @validate.`com.linkedin.dataset.rest.validator.DataPlatformValidator` = { }
+  platform: DataPlatformUrn
+
+  /**
+   * ML Model name e.g. {db}.{table}, /dir/subdir/{name}, or {name}
+   */
+  @validate.strlen = {
+    "max" : 500,
+    "min" : 1
+  }
+  name: string
+
+  /**
+   * Fabric type where ML Model belongs to or where it was generated.
+   */
+  origin: FabricType
```

Comment:
Does this Fabric refer to data centers? Just for my understanding, would models be trained per fabric?

Reply:
`FabricType` refers to the origin or environment. 
In most cases i don't think someone would train in every environment but they can attach the trained datasets (`DatasetUrn`) per platform, Model and Environment.